cov.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" ?>
+<coverage version="6.3.2" timestamp="1646702571660" lines-valid="56" lines-covered="47" line-rate="0.8393" branches-valid="22" branches-covered="16" branch-rate="0.7273" complexity="0">
```

Comment:
Do we need this in git?

Reply:
No, that was an artifact of an extension I have.